initial_posture: [0.0, 0.8, 1.3, 0.0, 0.0]
pre_pre: [0.0, 0.42, 1.12, 0.0, 0.0]
waypoint_pre: [0.0, 0.0, 0.03, 0.0, 0.0, 0.0]
waypoint_man: [0.0, 0.0, -0.08, 0.0, 0.0, 0.0]
waypoint_extract: [0.0, 0.0, 0.08, 0.0, 0.0, 0.0]
waypoint_insert: [0.0, 0.01, -0.09, 0.0, 0.0, 0.0]
